e2e/cypress/integration/api/iam/projects_api_ingestion.spec.ts

```diff
@@ -141,30 +152,82 @@ describeIfIAMV2p1('Ingestion project tagging', () => {
       }
     }
   ];
-  const complianceNodeId = uuidv4();
-  const clientRunsNodeId = uuidv4();
-  const reportId = uuidv4();
-  const nodeName = `${cypressPrefix}-${Cypress.moment().format('MMDDYYhhmm')}`;
-  const start = Cypress.moment().utc().subtract(3, 'day').startOf('day').format();
-  const end = Cypress.moment().utc().endOf('day').format();
+
+  const projectsWithEventRules = [
```

Comment:
Continuing on the same theme, strongly typing this composite object adds robustness:
```suggestion
  const projectsWithEventRules: ProjectAndRule[]  = [
```
Then you will need to define an interface:
```
interface ProjectAndRule {
    project: Project;
    rule: Rule;
  }
```
and apply imports:
```
import { Rule } from '../../../../../components/automate-ui/src/app/entities/rules/rule.model';
import { Project } from '../../../../../components/automate-ui/src/app/entities/projects/project.model';
```